docs/flow/reference/components/prometheus.relabel.md

```diff
@@ -0,0 +1,172 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/prometheus.relabel
+title: prometheus.relabel
+---
+
+# prometheus.relabel
+
+Prometheus metrics follow the [OpenMetrics](https://openmetrics.io/) format.
+Each time series is uniquely identified by its metric name, plus optional
+key-value pairs called labels. Each sample, i.e. a datapoint in the time
```

Comment:
Apparently it's recommended to avoid e.g. and i.e. because people tend to mix them up:

https://developers.google.com/style/abbreviations?hl=en#dont-use